### PR TITLE
python3Packages.ambiclimate: init at 0.2.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python library to communicate with ambiclimate.

https://github.com/Danielhiversen/pyAmbiclimate

This is a Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/home-assistant/component-packages.nix [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "almond" = ps: with ps; [ aiohttp-cors ]; # missing inputs: pyalmond
     "alpha_vantage" = ps: with ps; [ ]; # missing inputs: alpha_vantage
     "amazon_polly" = ps: with ps; [ boto3 ];
-    "ambiclimate" = ps: with ps; [ aiohttp-cors ]; # missing inputs: ambiclimate
+    "ambiclimate" = ps: with ps; [ aiohttp-cors ambiclimate ];
```
Comment (mweinelt) [inline pkgs/servers/home-assistant/component-packages.nix:28]: Please move this into a separate commit as you are modifying the home-assistant package.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 108233` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python37Packages.ambiclimate</li>
    <li>python38Packages.ambiclimate</li>
    <li>python39Packages.ambiclimate</li>
  </ul>
</details>

**Review by mweinelt — CHANGES_REQUESTED** [review]
Thanks for contributing, I have one tiny request, that seems to have not been mentioned on your previous contributions ot the home-assistant package over here.

**Review by mweinelt — APPROVED** [review]
